connector/src/main/java/org/geysermc/connector/network/UpstreamPacketHandler.java

```diff
@@ -111,10 +112,11 @@ private boolean couldLoginUserByName(String bedrockUsername) {
 
     @Override
     public boolean handle(SetLocalPlayerAsInitializedPacket packet) {
+        GeyserConfiguration config = bootstrap.getGeyserConfig();
         if (!session.isLoggedIn() && !session.isLoggingIn() && session.getConnector().getAuthType() == AuthType.ONLINE) {
             // TODO it is safer to key authentication on something that won't change (UUID, not username)
             if (!couldLoginUserByName(session.getAuthData().getName())) {
-                LoginEncryptionUtils.showLoginWindow(session);
+                LoginEncryptionUtils.showLoginWindow(session,AuthType.getByName(config.getRemote().getAuthType()));
```

Comment:
-.-

Reply:
This should be changed to use `GeyserConnector.getInstance().getAuthType()`